tsconfig.json

```diff
@@ -5,7 +5,8 @@
         "removeComments": false,
         "preserveConstEnums": true,
         "outDir": ".build",
-        "allowJs": true
+        "allowJs": true,
+        "target": "es2015"
```

Comment:
I did this so I could use `Set()`. I don't think any University students are going to be using browsers more than 4 years old, are they (all browsers supported the full es2015 spec by 2016).